bitnami/elasticsearch/templates/ingest/pdb.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and (include "elasticsearch.ingest.enabled" .) .Values.ingest.pdb.create }}
+apiVersion: {{ include "common.capabilities.policy.apiVersion" . }}
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "elasticsearch.ingest.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: ingest
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.ingest.pdb.minAvailable }}
+  minAvailable: {{ .Values.ingest.pdb.minAvailable }}
+  {{- end }}
+  {{- if .Values.ingest.pdb.maxUnavailable }}
+  maxUnavailable: {{ .Values.ingest.pdb.maxUnavailable }}
+  {{- end }}
+  selector: {{- include "common.labels.matchLabels" . | nindent 4 }}
```

Comment:
This part would be more correct to write it as follows:
```
selector:
    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
      app.kubernetes.io/component: ingest
```
This change should be made to all pdb.yaml files in this PR.